infra/pr_helper.py

```diff
@@ -223,6 +229,35 @@ def get_pull_request_number(self, commit):
       return None
     return pr_response.json()[0]['number']
 
+  def get_past_contributors(self, project_path):
+    """Returns a list of past contributors of a certain project."""
+    commits_response = requests.get(f'{BASE_URL}/commits?path={project_path}',
+                                    headers=self._headers)
+
+    if not commits_response.ok:
+      return []
+    commits = commits_response.json()
+    contributors: dict[str, bool] = {}
+    count = 0
+    for commit in commits:
+      login = commit['author']['login']
+      verified = commit['commit']['verification']['verified']
+      if login not in contributors:
+        contributors[login] = verified
+      if verified:
+        # Override previous verification bit.
+        contributors[login] = True
+      count += 1
+      if count >= COMMITS_LIMIT:
```

Comment:
nit: just use `for i, commit in enumerate(commits)` instead of keeping our own counter.